lib/queue_classic_plus/datadog.rb

```diff
@@ -2,8 +2,14 @@
 
 module QueueClassicDatadog
   def _perform(*args)
-    Datadog.tracer.trace('qc.job', service_name: 'qc.job', resource: "#{name}#perform") do |_|
-      super
+    if Gem.loaded_specs['ddtrace'].version >= '1'
+      Datadog::Tracing.trace('qc.job', service: 'qc.job', resource: "#{name}#perform") do |_|
```

Comment:
I wonder if when the app that includes queue classic plus has `ddtrace < 1`, will this generate exceptions for them for `Datadog::Tracing` not being defined?

Reply:
i opened a PR to test this against another of our apps and tests seem to be passing

Reply:
Good thinking! Presumably it's fine because the code is parsed but not executed for versions prior to 1.